### ISSUE TITLE
Bug Report by Brad - Attempt to disable an unimplemented interactable

### ISSUE THREAD

**Issue by Pustalorc** [Pustalorc/AutoTurnOff#4]
Provided description:
```
Error in MulticastDelegate PlayerDisconnected: System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> System.NotImplementedException: Interactable type: InteractableMannequin does not have an implemented wrapper!
  at Pustalorc.Plugins.AutoTurnOff.Interactables.InteractableWrapperHandler.GetInteractableWrapper (SDG.Unturned.Interactable interactable) [0x00047] in <16962a215cb841729f1ae95dd13cc335>:0
  at Pustalorc.Plugins.AutoTurnOff.ReduceLagPlugin.Disconnected (Rocket.Unturned.Player.UnturnedPlayer player) [0x0014f] in <16962a215cb841729f1ae95dd13cc335>:0
  at (wrapper managed-to-native) System.Reflection.MonoMethod.InternalInvoke(System.Reflection.MonoMethod,object,object[],System.Exception&)
  at System.Reflection.MonoMethod.Invoke (System.Object obj, System.Reflection.BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x00032] in <7ba07f088431485bb722f3b3373e87ee>:0
   --- End of inner exception stack trace ---
  at System.Reflection.MonoMethod.Invoke (System.Object obj, System.Reflection.BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x00048] in <7ba07f088431485bb722f3b3373e87ee>:0
  at System.Reflection.MethodBase.Invoke (System.Object obj, System.Object[] parameters) [0x00000] in <7ba07f088431485bb722f3b3373e87ee>:0
  at System.Delegate.DynamicInvokeImpl (System.Object[] args) [0x000e7] in <7ba07f088431485bb722f3b3373e87ee>:0
  at System.MulticastDelegate.DynamicInvokeImpl (System.Object[] args) [0x00008] in <7ba07f088431485bb722f3b3373e87ee>:0
  at System.Delegate.DynamicInvoke (System.Object[] args) [0x00000] in <7ba07f088431485bb722f3b3373e87ee>:0
  at Rocket.Core.Extensions.MulticastDelegateExtension.TryInvoke (System.MulticastDelegate theDelegate, System.Object[] args) [0x00019] in <466fa14d466749559a64a11ae8acbc60>:0
```

**Comment by Pustalorc** [comment]
This was also provided by Mr. P on discord:
```
[12/16/2019 4:55:49 PM] [Error] Error in MulticastDelegate PlayerDisconnected: System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> System.NotImplementedException: Interactable type: InteractableFire does not have an implemented wrapper!
  at Pustalorc.Plugins.AutoTurnOff.Interactables.InteractableWrapperHandler.GetInteractableWrapper (SDG.Unturned.Interactable interactable) [0x00047] in <16962a215cb841729f1ae95dd13cc335>:0 
  at Pustalorc.Plugins.AutoTurnOff.ReduceLagPlugin.Disconnected (Rocket.Unturned.Player.UnturnedPlayer player) [0x0014f] in <16962a215cb841729f1ae95dd13cc335>:0 
  at (wrapper managed-to-native) System.Reflection.MonoMethod.InternalInvoke(System.Reflection.MonoMethod,object,object[],System.Exception&)
  at System.Reflection.MonoMethod.Invoke (System.Object obj, System.Reflection.BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x00032] in <7ba07f088431485bb722f3b3373e87ee>:0 
   --- End of inner exception stack trace ---
  at System.Reflection.MonoMethod.Invoke (System.Object obj, System.Reflection.BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x00048] in <7ba07f088431485bb722f3b3373e87ee>:0 
  at System.Reflection.MethodBase.Invoke (System.Object obj, System.Object[] parameters) [0x00000] in <7ba07f088431485bb722f3b3373e87ee>:0 
  at System.Delegate.DynamicInvokeImpl (System.Object[] args) [0x000e7] in <7ba07f088431485bb722f3b3373e87ee>:0 
  at System.MulticastDelegate.DynamicInvokeImpl (System.Object[] args) [0x00008] in <7ba07f088431485bb722f3b3373e87ee>:0 
  at System.Delegate.DynamicInvoke (System.Object[] args) [0x00000] in <7ba07f088431485bb722f3b3373e87ee>:0 
  at Rocket.Core.Extensions.MulticastDelegateExtension.TryInvoke (System.MulticastDelegate theDelegate, System.Object[] args) [0x00019] in <466fa14d466749559a64a11ae8acbc60>:0
```

**Comment by Pustalorc** [comment]
Fixed with f01ea00fdd28338fe5a3fa28bb091d62514dc985
